### ISSUE TITLE
Python version is not consistent when built manually

### ISSUE THREAD

**Issue by lerouxrgd** [yahoojapan/NGT#63]
If we build `ngpty` manually (after building NGT C++ manually), the Python version is not consistent with the C++ version.

Building sdist is fine:

```sh
root@laptop:/NGT-1.10.0/python# python setup.py sdist --static-library
use the NGT static library
/usr/local/lib/python3.7/site-packages/setuptools/dist.py:454: UserWarning: Normalizing '1.10.0
' to '1.10.0'
  warnings.warn(tmpl.format(**locals()))
running sdist
running egg_info
writing ngt.egg-info/PKG-INFO
writing dependency_links to ngt.egg-info/dependency_links.txt
writing requirements to ngt.egg-info/requires.txt
writing top-level names to ngt.egg-info/top_level.txt
reading manifest file 'ngt.egg-info/SOURCES.txt'
writing manifest file 'ngt.egg-info/SOURCES.txt'
running check
creating ngt-1.10.0
creating ngt-1.10.0/ngt
creating ngt-1.10.0/ngt.egg-info
creating ngt-1.10.0/src
copying files to ngt-1.10.0...
copying README.md -> ngt-1.10.0
copying setup.py -> ngt-1.10.0
copying ngt/__init__.py -> ngt-1.10.0/ngt
copying ngt/base.py -> ngt-1.10.0/ngt
copying ngt.egg-info/PKG-INFO -> ngt-1.10.0/ngt.egg-info
copying ngt.egg-info/SOURCES.txt -> ngt-1.10.0/ngt.egg-info
copying ngt.egg-info/dependency_links.txt -> ngt-1.10.0/ngt.egg-info
copying ngt.egg-info/requires.txt -> ngt-1.10.0/ngt.egg-info
copying ngt.egg-info/top_level.txt -> ngt-1.10.0/ngt.egg-info
copying src/ngtpy.cpp -> ngt-1.10.0/src
Writing ngt-1.10.0/setup.cfg
Creating tar archive
removing 'ngt-1.10.0' (and everything under it)
```

Installing deps is fine:

```sh
root@laptop:/NGT-1.10.0/python# pip install numpy pybind11
Collecting numpy
  Using cached numpy-1.18.3-cp37-cp37m-manylinux1_x86_64.whl (20.2 MB)
Collecting pybind11
  Using cached pybind11-2.5.0-py2.py3-none-any.whl (296 kB)
Installing collected packages: numpy, pybind11
Successfully installed numpy-1.18.3 pybind11-2.5.0
```

However, when installing `dist/ngt-1.10.0.tar.gz` it will always use version `1.8.0`:

```sh
root@laptop:/NGT-1.10.0/python# pip install dist/ngt-1.10.0.tar.gz
Processing ./dist/ngt-1.10.0.tar.gz
Requirement already satisfied: numpy in /usr/local/lib/python3.7/site-packages (from ngt==1.8.0) (1.18.3)
Requirement already satisfied: pybind11 in /usr/local/lib/python3.7/site-packages (from ngt==1.8.0) (2.5.0)
Building wheels for collected packages: ngt
  Building wheel for ngt (setup.py) ... done
  Created wheel for ngt: filename=ngt-1.8.0-cp37-cp37m-linux_x86_64.whl size=2163885 sha256=d99c8d886f4cf683b1baa6d1746eccc8b71f0b97f974e3347296e3d4097abb44
  Stored in directory: /root/.cache/pip/wheels/f7/49/42/c4dfb80dde389e6625f9ce343ea83054debfe23687ec21e49b
Successfully built ngt
Installing collected packages: ngt
Successfully installed ngt-1.8.0
```
I think this is due to the hardcoded `version = '1.8.0'` in `setup.py`.

Moreover there is no variable `ngt.__version__` to check what version we are using in Python, this is a bit confusing.

**Comment by masajiro** [comment]
Thank you for your detailed information.
I define the version 1.8.0 as ngtpy **API** version.  However, I recognize that it is confusing because another user also pointed it out. I will fix the issues you pointed out soon.

**Comment by masajiro** [comment]
I have released v1.11.0 fixing this issue. If you have any problems, please let me know.
